commit f02070cb4b603b36ff2a3343c2429b279c9f8084
Author: Anson Huang <anson.huang@nxp.com>
Date:   Sat Dec 29 10:01:18 2018 +0000

    ARM: dts: imx6sx: correct backward compatible of gpt
    
    [ Upstream commit ba0f4560526ba19300c07ed5a3c1df7592815dc6 ]
    
    i.MX6SX has same GPT type as i.MX6DL, in GPT driver, it uses
    below TIMER_OF_DECLARE, so the backward compatible should be
    "fsl,imx6dl-gpt", correct it.
    
    TIMER_OF_DECLARE(imx6sx_timer, "fsl,imx6sx-gpt", imx6dl_timer_init_dt);
    
    Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
    Signed-off-by: Shawn Guo <shawnguo@kernel.org>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 844caa39364f..50083cecc6c9 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -462,7 +462,7 @@
 			};
 
 			gpt: gpt@2098000 {
-				compatible = "fsl,imx6sx-gpt", "fsl,imx31-gpt";
+				compatible = "fsl,imx6sx-gpt", "fsl,imx6dl-gpt";
 				reg = <0x02098000 0x4000>;
 				interrupts = <GIC_SPI 55 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6SX_CLK_GPT_BUS>,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1067_f02070cb4b603b36ff2a3343c2429b279c9f8084.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1067_f02070cb4b603b36ff2a3343c2429b279c9f8084.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
